flink-runtime/src/main/java/org/apache/flink/runtime/client/DuplicateJobSubmissionException.java

```diff
@@ -35,10 +35,24 @@ public static DuplicateJobSubmissionException ofGloballyTerminated(JobID jobId)
         return new DuplicateJobSubmissionException(jobId, true);
     }
 
+    public static DuplicateJobSubmissionException ofGloballyTerminatedAndCleaned(JobID jobId) {
+        return new DuplicateJobSubmissionException(
+                jobId,
+                true,
+                "There's already a job with the given JobID '"
+                        + jobId
+                        + "' listed as globally-terminated and cleaned. No further actions required.");
```

Comment:
I'm not sure if this is sufficient.

It says "No further actions required", but I'm not sure whether this refers to Flink ("I don't need to run anything") or the user ("I can ignore this error").
Then, let's say a user really wants to run the job anyway; how do they achieve that? (This is particularly important because ZERO_JOB_ID exists)